docs/topics/serialization.txt

```diff
@@ -268,6 +268,15 @@ work::
                 return force_text(obj)
             return super(LazyEncoder, self).default(obj)
 
+By default, :class:`~django.core.serializers.json.DjangoJSONEncoder` is used
```

Comment:
You need to add `.. class:: django.core.serializers.json.DjangoJSONEncoder` somewhere if you want the link to work.